db/import_listener.go

```diff
@@ -43,12 +44,27 @@ func (il *importListener) StartImportFeed(bucket base.Bucket, dbStats *base.DbSt
 	il.bucketName = bucket.GetName()
 	il.database = Database{DatabaseContext: dbContext, user: nil}
 	il.stats = dbStats.Database()
+	collections := make(map[string][]string)
```

Comment:
This was called `scopes` as a map of scopes to collections in the above functions - maybe rename for consistency